### ISSUE TITLE
Documentation mistake - using PostgreSQL on host is possible

### ISSUE THREAD

**Issue by Samonitari** [spantaleev/matrix-docker-ansible-deploy#1679]
**Describe the bug**
[Documentation on external PostgreSQL setup](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-external-postgres.md) is a bit too conservative: I think using a non-containerized database **is** possible:
You could connect to host's PostgreSQL through a socket(s) mounted to the containers.
Playbook would need some tweaking so the systemd-services would have additional `--mount` argument in `ExecStart` command, but that doesn't seem too difficult...

**Additional context**
I'll try to convert my setup as mentioned, and leave some comments if successful.

**Comment by Samonitari** [comment]
I have invested some time, but guess what: you were correct :smile: !

Even before reporting this issue, I've migrated to an external, but containerized postgresql (I've migrated my existing matrix databases to that container, changed connection settings for all playbook managed servives, etc.). Reason is, my server is quite constrained, so I wanted to use a single database container.
That worked fine-ish (server's databases of nextcloud and postfix in turn were migrated to postgresql from mariadb, it was *"fun"* - I seem to not learn the lesson that initial setup choices should be made **with great care**...), and I see the docs are extended to mention downsides.

But a non-containerized external database is **mostly** impossible. I've thought that with sockets it would be easy, but ma1sd cannot use unix sockets, **because [Java](https://stackoverflow.com/a/25920897/4428260)** (it may be possible, but would need contribution to ma1sd, which is not worth the effort.

Thanks for your time, I am closing this issue, as ma1sd is an important component installed by default!
